app/src/main/java/com/cornellappdev/transit/ui/screens/RouteScreen.kt

```diff
@@ -0,0 +1,164 @@
+                        imageVector = Icons.Outlined.KeyboardArrowLeft,
+                        contentDescription = ""
+                    )
+                }
+            }
+
+        )
+
+        Divider(thickness = 1.dp, color = DividerGrey)
+        Row(
+            modifier = Modifier
+                .padding(horizontal = 20.dp, vertical = 12.dp),
+            horizontalArrangement = Arrangement.spacedBy(16.dp),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            // From-to column
+            Column(verticalArrangement = Arrangement.spacedBy(24.dp)) {
+                Text(
+                    text = "From",
+                    color = PrimaryText,
+                    fontFamily = sfProDisplayFamily,
+                    fontWeight = FontWeight.Normal
+                )
+                Text(
+                    text = "To",
+                    color = PrimaryText,
+                    fontFamily = sfProDisplayFamily,
+                    fontWeight = FontWeight.Normal
+                )
+            }
+
+            Icon(
+                painter = painterResource(id = R.drawable.route_icon),
+                contentDescription = "",
+                modifier = Modifier.size(width = 16.dp, height = 54.dp),
+                tint = Color.Unspecified
+            )
+
+            //TODO user should be able to enter location/destination
+            Column(verticalArrangement = Arrangement.spacedBy(12.dp)) {
+                Box(
+                    modifier = Modifier
+                        .background(color = DividerGrey, shape = RoundedCornerShape(8.dp))
+                        .fillMaxWidth(0.9f)
+                ) {
+                    Text(
+                        text = "Current Location",
+                        modifier = Modifier.padding(horizontal = 12.dp, vertical = 6.dp),
+                        color = PrimaryText,
+                        fontFamily = sfProDisplayFamily,
+                        fontWeight = FontWeight.Normal
+                    )
+                }
+                Box(
+                    modifier = Modifier
+                        .background(color = DividerGrey, shape = RoundedCornerShape(8.dp))
+                        .fillMaxWidth(0.9f)
+                ) {
+                    Text(
+                        text = "Destination",
+                        modifier = Modifier.padding(horizontal = 12.dp, vertical = 6.dp),
+                        fontFamily = sfProDisplayFamily,
+                        fontWeight = FontWeight.Normal
+                    )
+                }
+            }
+
+            //TODO this should probably be an IconButton that swaps current/dest
+            Icon(
+                painter = painterResource(id = R.drawable.swap_icon),
+                contentDescription = "",
+                modifier = Modifier.size(width = 20.dp, height = 20.dp),
+                tint = Color.Unspecified
+            )
```

Comment:
We need to make the swap icon an SVG. The difference is that a png is a matrix of pixels whereas an SVG is a math definition for how to construct a shape. That means it will NEVER be blurry since you can rescale it to any size and have the formula do its thang.